src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -326,6 +346,10 @@ void BasePlaylistFeature::slotCreatePlaylist() {
 void BasePlaylistFeature::slotDeletePlaylist() {
     //qDebug() << "slotDeletePlaylist() row:" << m_lastRightClickedIndex.data();
     int playlistId = playlistIdFromIndex(m_lastRightClickedIndex);
+    QModelIndex nextIndex =
+            m_lastRightClickedIndex.sibling(m_lastRightClickedIndex.row() + 1,
+                    m_lastRightClickedIndex.column());
+    int nextId = playlistIdFromIndex(nextIndex);
     if (playlistId == -1) {
```

Comment:
A similar use case. Check the value playlistId immediately and otherwise early return as soon as possible.

Reply:
nextId can be -1, it's not an error case

Reply:
I didn't mention nextId.

Reply:
fixed